### ISSUE TITLE
V3 Release schedule

### ISSUE THREAD

**Issue by exsilium** [xtermjs/xterm.js#1116]
Not to rush you guys or anything but I was wondering if you have a release schedule in mind when it comes to the 3.0? Perhaps some indication or blocker issue list that needs to be done prior to a release?

Thanks in advance and Cheers! 🍻

**Comment by parisk** [comment]
Hi @exsilium!

It should be out within a week or two (max). Most of the stuff are in place, but we would like to include also #1018 and a new website/logo maybe in the release.

We are working on these right now and in a few days everything should be ready to launch!

**Comment by parisk** [comment]
Just an update here that the new website or logo won't be blocking the release, but they are being worked in parallel 😄.

**Comment by Tyriar** [comment]
The logo is in (https://github.com/xtermjs/xterm.js/pull/1125, https://github.com/xtermjs/xtermjs-branding), remaining tasks are:

- Closing/deferring remaining issues on the milestone https://github.com/xtermjs/xterm.js/issues?q=is%3Aopen+is%3Aissue+milestone%3A3.0.0
- Merging/deferring remaining PRs on the milestone https://github.com/xtermjs/xterm.js/pulls?q=is%3Aopen+is%3Apr+milestone%3A3.0.0

**Comment by grunmin** [comment]
any blocker issue?😕

**Comment by Tyriar** [comment]
I've been really busy which has blocked the release (vacation/moving house). It might still be another 2 weeks before the release.

**Comment by exsilium** [comment]
Happy New Year! 🎆 🍻 😉

**Comment by Tyriar** [comment]
We're so close, closing this in favor of https://github.com/xtermjs/xterm.js/issues/1184, check that for the plan 😄
